src/hotspot/share/gc/g1/g1GCPhaseTimes.cpp

```diff
@@ -103,6 +103,7 @@ G1GCPhaseTimes::G1GCPhaseTimes(STWGCTimer* gc_timer, uint max_gc_threads) :
   _gc_par_phases[Other] = new WorkerDataArray<double>("Other", "GC Worker Other (ms):", max_gc_threads);
   _gc_par_phases[MergePSS] = new WorkerDataArray<double>("MergePSS", "Merge Per-Thread State (ms):", max_gc_threads);
   _gc_par_phases[RestoreRetainedRegions] = new WorkerDataArray<double>("RestoreRetainedRegions", "Restore Retained Regions (ms):", max_gc_threads);
+  _gc_par_phases[RemoveSelfForwardsInChunks] = new WorkerDataArray<double>("RemoveSelfForwardsInChunks", "Remove Self Forwards In Chunks (ms):", max_gc_threads);
```

Comment:
Is the "In Chunks" part really interesting to users? We do mention chunks below so the info is there. 

And if we could come up with a more user friendly name than "Remove Self Forwards" that would probably be good as well. Some thing like "Clean/Fix Failed Regions" maybe.

Reply:
I can see your point about the "InChunks" postfix; however I would prefer to keep the phase meaningful wrt to what it does. Neither "Remove Self Forwards" or "Fix Failed Regions" helps end users in a big way, but the former helps at least us understand at a glance what this phase does without needing to translate "Fix Failed Regions" to "this phase removes the self forwards".

Reply:
Sure, that's a fair point. Logging at this level requires some knowledge to be fully meaningful as well.